opm/simulators/flow/ReservoirCouplingSpawnSlaves.cpp

```diff
@@ -168,12 +193,14 @@ getSlaveArgv_(
     return slave_argv;
 }
 
+template <class Scalar>
 std::pair<std::vector<char>, std::size_t>
-ReservoirCouplingSpawnSlaves::
+ReservoirCouplingSpawnSlaves<Scalar>::
 getMasterGroupNamesForSlave_(const std::string &slave_name) const
 {
     // For the given slave name, get all pairs of master group names and slave group names
     // Serialize the data such that it can be sent over MPI in one chunk
+    // NOTE: The order of the names in the vector is important, as the slaves will
```

Comment:
Please complete the sentence.
Is there a similar note in the class documentation of the type of rescoup_? That is where people using the class would look.